dynamics/test/Advection_test.cpp

```diff
@@ -294,16 +296,28 @@ TEST_CASE("Advection")
 }
 TEST_CASE("Distorted Mesh")
 {
-    std::array<std::array<double, 6>, 3> exact
-        = { std::array<double, 6>(
-                { 5.0958748236875594e-02, 4.8461087243594887e-02, 4.3918572621094686e-02,
-                    3.7038043078562323e-02, 2.8334464207979679e-02, 1.9666196904181983e-02 }),
-              std::array<double, 6>(
-                  { 3.2033423984965226e-02, 1.5639052766007147e-02, 6.7926997203524983e-03,
-                      2.7369916393700151e-03, 9.2109964949992139e-04, 2.5128064874203957e-04 }),
-              std::array<double, 6>(
-                  { 1.1830071142946669e-02, 4.9207680503709564e-03, 1.5831512504162868e-03,
-                      3.8869595113351363e-04, 6.7162895595772516e-05, 7.5853786764529606e-06 }) };
+    // std::array<std::array<double, 6>, 3> exact
+    //     = { std::array<double, 6>(
+    //             { 5.0958748236875594e-02, 4.8461087243594887e-02, 4.3918572621094686e-02,
+    //                 3.7038043078562323e-02, 2.8334464207979679e-02, 1.9666196904181983e-02 }),
+    //           std::array<double, 6>(
+    //               { 3.2033423984965226e-02, 1.5639052766007147e-02, 6.7926997203524983e-03,
+    //                   2.7369916393700151e-03, 9.2109964949992139e-04, 2.5128064874203957e-04 }),
+    //           std::array<double, 6>(
+    //               { 1.1830071142946669e-02, 4.9207680503709564e-03, 1.5831512504162868e-03,
+    //                   3.8869595113351363e-04, 6.7162895595772516e-05, 7.5853786764529606e-06 })
+    //                   };
```

Comment:
Consider removing these old values, as they can be obtained from the file history

Reply:
👍